File: Y/lv0/짝수홀수개수.js
function solution(num_list) {
    var answer = [];

    for(i=0; i<num_list.length; i++){
        if(num_list[i] % 2 == 0){
            answer[0] += 1;
            // answer[i] = s;
        }else{
            answer[1] += 1;
            console.log("else")
            // answer[i] = i
        }
    }
    return answer;
}

File: L/lv0/짝수홀수개수.java
package lv0;

public class 짝수홀수개수 {
    class Solution {
        public int[] solution(int[] num_list) {
            int[] answer = new int[2];
            for(int i =0; i<num_list.length; i++){
                if(num_list[i]%2 == 0){
                    System.out.println(num_list[i]);
                    answer[0] += 1;
                }else {
                    System.out.println(num_list[i]);
                    answer[1] += 1;
                }
            }
            return answer;
        }
    }
}
